dbt/adapters/athena/impl.py

```diff
@@ -357,7 +358,8 @@ def _s3_path_exists(self, s3_bucket: str, s3_prefix: str) -> bool:
         response = s3_client.list_objects_v2(Bucket=s3_bucket, Prefix=s3_prefix)
         return True if "Contents" in response else False
 
-    def _join_catalog_table_owners(self, table: agate.Table, manifest: Manifest) -> agate.Table:
+    @staticmethod
+    def _join_catalog_table_owners(table: agate.Table, manifest: Manifest) -> agate.Table:
```

Comment:
If you are using `staticmethod` I would be in favour of changing the call to this function to reflect that now it is a static method.

Reply:
@Jrmyy I've removed the change coz it does not belong to current PR. However could you specify which call to static method do you mean ?